WebApi/WebApi/Controllers/CommentController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using WebApi.Models.DTO;
+using WebApi.Services.Services_Interfaces;
+
+namespace WebApi.Controllers
+{
+    [ApiController]
+    [Route("api/comment")]
+    public class CommentController
+    {
+        private readonly ICommentService _commentService;
+        private readonly ILogger<CommentController> _logger;
+        public CommentController(ILogger<CommentController> logger, ICommentService commentService)
+        {
+            _commentService = commentService;
+            _logger = logger;
+        }
+        [HttpGet]
+        public IQueryable<CommentDTO> GetAll([FromHeader] int userId)
+        {
+            return _commentService.GetAll(userId);
+        }
+        [HttpGet("{id}")]
+        public CommentDTO GetById([FromRoute] int id, [FromHeader] int userId)
+        {
+            return _commentService.GetById(id,userId);
+        }
+       //Nw jak w post ma byc juz id 
+        [HttpPost]
+        public async Task<CommentDTO> AddComment([FromBody] CommentDTO comment, [FromHeader] int userId)
+        {
+            return await _commentService.AddCommentAsync(userId,comment);
```

Comment:
Według dokumentacji, przy tworzeniu nowego komentarza, podajemy userID w header,
natomiast w body ma być content oraz postID (id postu do którego jest ten komentarz).

Ma to być zrobione tak, że robisz DTO które ma tylko te dwa pola: content, postID
Następnie do CommentsService przekazujesz to DTO oraz userID i tam mapujesz to na CommentsDTO i dodajesz datę aktualną w maperze itp